http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20200123_populate_date_accessed.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="leonardo" author="qi" id="populate_date-accessed">
+        <sql>UPDATE RUNTIME_CONFIG r set r.dateAccessed = NOW() where r.dateAccessed is NULL</sql>
```

Comment:
Need to then set a not-null constraint on the column

Reply:
updated